src/coins.ts

```diff
@@ -733,6 +733,49 @@ const basicCoins: BasicCoin[] = [
     },
   },
 
+  // USDC
+  {
+    key: CoinKey.BUSD,
+    name: CoinKey.BUSD,
+    logoURI:
+      'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/binance/assets/BUSD-BD1/logo.png',
+    verified: true,
+    chains: {
+      [ChainId.ETH]: {
+        address: '0x4Fabb145d64652a948d72533023f6E7A623C7C53',
+        decimals: 18,
+      },
+      [ChainId.BSC]: {
+        address: '0xe9e7cea3dedca5984780bafc599bd69add087d56',
+        decimals: 18,
+      },
+      [ChainId.POL]: {
+        address: '0xdAb529f40E671A1D4bF91361c21bf9f0C9712ab7',
+        decimals: 18,
+      },
+      [ChainId.AVA]: {
+        address: '0x9C9e5fD8bbc25984B178FdCE6117Defa39d2db39',
+        decimals: 18,
+      },
+      [ChainId.SOL]: {
+        address: '33fsBLA8djQm82RpHmE3SuVrPGtZBWNYExsEUeKX1HXX',
+        decimals: 18,
+      },
+      [ChainId.FUS]: {
+        address: '0x6a5F6A8121592BeCd6747a38d67451B310F7f156',
+        decimals: 18,
+      },
+      [ChainId.VEL]: {
+        address: '0xc111c29a988ae0c0087d97b33c6e6766808a3bd3',
+        decimals: 18,
+      },
+      [ChainId.ONE]: {
+        address: '0xe176ebe47d621b984a73036b9da5d834411ef734',
+        decimals: 18,
+      },
```

Comment:
Please add it to all our chains where it exists, e.g.
https://blockscout.com/xdai/mainnet/token/0xdd96B45877d0E8361a4DDb732da741e97f3191Ff/token-transfers
https://arbiscan.io/token/0x52095f518048fafcfba0ec9f93962df449bc604e
https://optimistic.etherscan.io/token/0x9c9e5fd8bbc25984b178fdce6117defa39d2db39